interview-150/14.ts

```diff
@@ -0,0 +1,21 @@
+function longestCommonPrefix(strs: string[]): string {
+    let size = strs.length
+
+    if (size === 0) {
+        return "0"
+    } else if (size === 1) {
+        return strs[0]
+    }
```

Comment:
Note: the `else if` is technically unnecessary because if `if` is true, it returns.